adamos204/src/main/java/dao/UserDao.java

```diff
@@ -0,0 +1,112 @@
+package dao;
+
+import model.User;
+import org.mindrot.jbcrypt.BCrypt;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class UserDao {
+
+    private static final String CREATE_USER_QUERY = "INSERT INTO users(email, username, password) VALUES (?, ?, ?)";
+    private static final String SELECT_ALL_USERS_QUERY = "SELECT * FROM users";
+    private static final String SELECT_USER_QUERY = "SELECT * FROM users WHERE id = ?";
+    private static final String UPDATE_USER_QUERY = "UPDATE users SET email = ?, username = ?, password= ? WHERE id = ?";
+    private static final String DELETE_USER_QUERY = "DELETE FROM users WHERE id = ?";
+
+    private String hashPasswd(String passwd) {
```

Comment:
IMPROVE: pro moznost pouziti mimo dao

```suggestion
public String hashPasswd(String passwd) {
```